src/arangodb/storage.py

```
@@ -4,19 +4,28 @@
 This module provides a complete ArangoDB storage component that implements the Storage
 protocol with full Sequential-ISNE support, real database operations, and comprehensive
 features for graph-enhanced RAG systems.
+
+NOTE: Async methods are implemented as synchronous wrappers using asyncio.run_in_executor.
+      True async support with native async database drivers is planned for a future release.
 """
 
+
 import logging
 import json
 from typing import Dict, Any, List, Optional, Union, Iterator, Tuple
```

Ruff (0.11.9)

`typing.Iterator` imported but unused

Remove unused import: `typing.Iterator`

(F401)